get_data/get_data.py

```diff
@@ -1,3 +1,25 @@
 """
 WIT oil price data is acquired from the Quandl API for use by the data_analysis module.
 """
+import numpy as np
+import matplotlib.pyplot as plt
+import quandl
+from datetime import datetime
+from statsmodels.tsa.arima_model import ARIMA
+import pandas as pd
+import os
+
+from pandas.plotting import register_matplotlib_converters
+
```

Comment:
Could you remove the unused imports here please? think you only need quandl?

Reply:
yeah will do :)